src/Controls/src/Core/Handlers/Items/iOS/StructuredItemsViewController.cs

```diff
@@ -239,5 +260,15 @@ internal void UpdateLayoutMeasurements()
 			if (_footerViewFormsElement != null)
 				HandleFormsElementMeasureInvalidated(_footerViewFormsElement);
 		}
+
+		private void HeaderView_LayoutChanged(object sender, EventArgs e)
```

Comment:
```suggestion
		void HeaderViewLayoutChanged(object sender, EventArgs e)
```

We don't use the private and try avoid using _ on method naming

Reply:
Fixed